src/app/reducers/disk-offerings/redux/disk-offerings.reducers.ts

```diff
@@ -113,3 +116,17 @@ export const getAvailableOfferings = createSelector(
     return [];
   },
 );
+
+export const isDiskOfferingAvailableByResources = (minSize: number) =>
+  createSelector(
+    fromAuth.getUserAccount,
+    fromVMs.getVmFormState,
+    (account, state): boolean => {
+      if (!isTemplate(state.template) && state.diskOffering) {
+        const storageAvailability = account.primarystorageavailable;
```

Comment:
Can we use selectStorageAvailable here instead of accessing account.primarystorageavailable?